dist/routable-component.js

```diff
@@ -124,7 +124,7 @@ var routableComponent =
 	
 	
 	  (0, _createClass3.default)(RoutableComponent, [{
-	    key: 'register_components',
+	    key: 'registerComponents',
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->